client/question/services/EventHandlerService.js

```diff
@@ -28,9 +59,15 @@ tie.factory('EventHandlerService', [
       createSessionPauseEvent: function(sessionId) {
         if (ServerHandlerService.doesServerExist()) {
           var data = {
-            sessionId: sessionId
+            sessionId: sessionId,
+            createdMsec: (new Date()).getTime()
+          };
+          var event = {
```

Comment:
Apparently `event` is a global value in IE (that's why you see it as blue when looking at it on github). It might be better to use evt instead, here and below.

Reply:
Huh. I'll just do the other thing you suggested.